src/encoded/static/components/item-pages/FileView.js

```diff
@@ -0,0 +1,173 @@
+    const breadcrumbs = [
+        { display_title: 'Home', href: '/' },
+        { display_title: 'Data' },
+        { display_title: 'Bechmarking Data' },
+        { display_title: context?.dataset?.toUpperCase() || '' },
+    ];
+
+    return (
+        <div className="file-view-title container-wide">
+            <nav className="file-view-title-navigation">
+                <ul className="breadcrumb-list">
+                    {breadcrumbs.map(({ display_title, href }, i, arr) => {
+                        return (
+                            <li className="breadcrumb-list-item" key={i}>
+                                <a
+                                    className={
+                                        'breadcrumb-list-item-link' +
+                                        (href ? '' : ' no-link')
+                                    }
+                                    href={href}>
+                                    {display_title}
+                                </a>
+                                {i < arr.length - 1 ? (
+                                    <i className="icon icon-fw icon-angle-right fas"></i>
+                                ) : null}
+                            </li>
+                        );
+                    })}
+                </ul>
+            </nav>
+            <h1 className="file-view-title-text">{context.display_title}</h1>
+        </div>
+    );
+};
+
+// Header component containing high-level information for the file item
+const FileViewHeader = (props) => {
+    const { context, session } = props;
+    const { accession, status, description } = context;
+    const selectedFile = new Map([[context['@id'], context]]);
+
+    return (
+        <div className="file-view-header">
+            <div className="data-group data-row header">
+                <h1 className="header-text">File Overview</h1>
+                <SelectedItemsDownloadButton
+                    id="download_tsv_multiselect"
+                    className="btn btn-primary btn-sm mr-05 align-items-center download-file-button"
+                    session={session}
+                    selectedItems={selectedFile}
+                    disabled={false}
+                    analyticsAddItemsToCart>
+                    <i className="icon icon-download fas mr-07" />
+                    Download File
+                </SelectedItemsDownloadButton>
+            </div>
+            <div className="data-group data-row">
+                <div className="datum">
+                    <span className="datum-title">File Accession </span>
+                    <span className="vertical-divider">|</span>
+                    <span>
+                        <b className="accession">{accession}</b>
+                    </span>
+                </div>
+                <div className="datum right-group">
+                    <div className="status-group" data-status={status}>
+                        <i className="icon icon-circle fas"></i>
```

Comment:
I think aria-label is not necessary here, since it is non-interactive/static and not focusable. It does seem to lack some context without a proper "status" header, but it is read right after the file accession which I think does the job.